utils/job_submitters/oar.py

```diff
@@ -19,12 +20,12 @@ def __init__(self, **kwargs):
         ##host specific settings
         if self.host == 'gricad':
             p = subprocess.run("hostname", capture_output=True, text=True)
-            if 'dahu' in p.stdout: ##TODO: what is the typical hostname for compute nodes? change the statement here accordingly.
+            if p.stdout.replace('\n', '').replace(' ', '') in ['dahu-oar3', 'f-dahu']:
```

Comment:
gridcad only has two nodes that support job submission, and they're 'dahu-oar3' and 'f-dahu'